Yeta-Wrench.md

```diff
@@ -1 +1,14 @@
-![](http://loenwind.info/eio/Yeta_Wrench.png)
\ No newline at end of file
+# Yeta Wrench
+![](http://loenwind.info/eio/Yeta_Wrench.png)
+
+The Yeta Wrench is a tool that has a lot of uses:
+
+* Conduit Facades are hidden while the Yeta Wrench is held
```

Comment:
facades is one of the display mode (see last bullet point). when the display mode is set to facades, facades are shown and can be shift-r-click removed. Can you adapt it to reflect this? thx